.cirrus.yml

```diff
@@ -108,19 +97,33 @@ task:
           # has been fixed, this can be removed and there can just be a single
           # run with --require-excerpts and no exclusions.
           - ./script/tool_runner.sh readme-check --require-excerpts --exclude=script/configs/temp_exclude_excerpt.yaml
-        license_script: dart $PLUGIN_TOOL license-check
-        dependabot_script: dart $PLUGIN_TOOL dependabot-check
-    - name: federated_safety
-      # This check is only meaningful for PRs, as it validates changes
-      # rather than state.
-      only_if: $CIRRUS_PR != ""
-      script: ./script/tool_runner.sh federation-safety-check
+        dependabot_script: $PLUGIN_TOOL_COMMAND dependabot-check
+        version_script:
+          # For pre-submit, pass the PR labels to the script to allow for
+          # check overrides.
+          # For post-submit, ignore platform version breaking version changes
+          # and missing version/CHANGELOG detection since the labels aren't
+          # available outside of the context of the PR.
+          - if [[ $CIRRUS_PR == "" ]]; then
+          -   ./script/tool_runner.sh version-check --ignore-platform-interface-breaks
+          - else
+          -   ./script/tool_runner.sh version-check --check-for-missing-changes --pr-labels="$CIRRUS_PR_LABELS"
+          - fi
+        publishable_script: ./script/tool_runner.sh publish-check --allow-pre-release
+        federated_safety_script:
+          # This check is only meaningful for PRs, as it validates changes
+          # rather than state.
+          - if [[ $CIRRUS_PR == "" ]]; then
```

Comment:
This was cleaner as an `only_if` rather than a conditional no-op, but that only operates at the task level, and the overhead of the task takes much longer than the step itself so keeping it separate just didn't make sense.